pkgs/development/python-modules/pgpy/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cryptography
+, pyasn1
+, six
+, singledispatch
+}:
+
+buildPythonPackage rec {
+  pname = "PGPy";
```

I think we're normalizing `pname` now so this should be
```suggestion
  pname = "pgpy";
```